scout/demo/cancer_coverage_qc_report.html

```diff
@@ -0,0 +1,307 @@
+<!DOCTYPE html>
+<!--Report template and solution from: https://github.com/NationalGenomicsInfrastructure/ngi_reports-->
```

Comment:
Super with an example! NGI? Does this particular one have a history to document? 😊

Reply:
It is from this PR on BALSAMIC: https://github.com/Clinical-Genomics/BALSAMIC/pull/543

And template (CSS, table format, etc) taken from https://github.com/NationalGenomicsInfrastructure/ngi_reports/blob/master/data/report_templates/project_summary.html with ok from authors. :-)